internal/operators/ocs/ocs_operator_test.go

```diff
@@ -17,31 +17,31 @@ var _ = Describe("Ocs Operator Test GetHostRequirements", func() {
 		ctx      = context.TODO()
 		operator = NewOcsOperator(common.GetTestLog())
 		master1  = &models.Host{Role: models.HostRoleMaster,
-			Inventory: Inventory(&InventoryResources{Cpus: 12, Ram: 32 * conversions.GB,
+			Inventory: Inventory(&InventoryResources{Cpus: 12, Ram: 32 * conversions.GiB,
 				Disks: []*models.Disk{
 					{SizeBytes: 20 * conversions.GB, DriveType: "HDD"},
 					{SizeBytes: 40 * conversions.GB, DriveType: "SSD"},
 				}})}
 		master2 = &models.Host{Role: models.HostRoleMaster, Inventory: Inventory(&InventoryResources{Cpus: 12, Ram: 32})}
```

Comment:
Does this need the conversions.GiB factor as well?

Reply:
Yes, I will do that